yarn.lock

```diff
@@ -2500,6 +2500,11 @@
     npmlog "^6.0.2"
     write-file-atomic "^4.0.1"
 
+"@littlespoon/theme@1.13.0":
+  version "1.13.0"
+  resolved "https://registry.yarnpkg.com/@littlespoon/theme/-/theme-1.13.0.tgz#c364f21b260919bfc2192f252a2fffa759700dd8"
+  integrity sha512-v62FyjIKMyvOCMAMZU20Iz4K0Mom5f46RWvdhFJ1X4jMPrRi8QBjePqAZ6WM9jdv26ZMPSRI/6OguBZfCNLB4g==
+
```

Comment:
Remove. This is caused by https://github.com/little-spoon-dev/design-system/pull/584/files#r949771408